src/main/java/gregtech/asm/GregTechTransformer.java

```diff
@@ -120,7 +121,8 @@ public byte[] transform(String name, String transformedName, byte[] basicClass)
                 ClassReader classReader = new ClassReader(basicClass);
                 ClassWriter classWriter = new ClassWriter(0);
 
-                ModContainer container = Loader.instance().getIndexedModList().get("nuclearcraft");
+                // fix NC recipe compat different depending on overhaul vs underhaul
+                ModContainer container = Loader.instance().getIndexedModList().get(GTValues.MODID_NC);
```

Comment:
Do we need to null check this `get` call in case NC is not in the mod list?

Reply:
We shouldn't need to here, since this case for the switch statement will only execute when the target class for NC is passed into the transformer, and is therefore loaded/present.